src/DebugBar/DataCollector/ExceptionsCollector.php

```diff
@@ -29,6 +31,9 @@ public function addException(Exception $e)
     {
         $this->exceptions[] = $e;
         if ($this->chainExceptions && $previous = $e->getPrevious()) {
+            if ($previous instanceof Error) {
```

Comment:
I disagree, I think this is more readable and more explicitly states the purpose of the code.